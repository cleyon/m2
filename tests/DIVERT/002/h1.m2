@sequence h1count
@newcmd h1{h1_text}
@divert 9
@sequence h1count next
<li><a href="#H1_@h1count@">@h1_text@</a></li>
@divert 1
<h1 id="H1_@h1count@">@h1_text@</h1>
@divert
@endcmd
@@
@h1{First header}
@h1{Second header}
@h1{Third header}
<strong>Table of contents</strong>
<ol>
@undivert 9
</ol>
@undivert 1
