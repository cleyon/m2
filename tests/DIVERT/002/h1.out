<strong>Table of contents</strong>
<ol>
<li><a href="#H1_1">First header</a></li>
<li><a href="#H1_2">Second header</a></li>
<li><a href="#H1_3">Third header</a></li>
</ol>
<h1 id="H1_1">First header</h1>
<h1 id="H1_2">Second header</h1>
<h1 id="H1_3">Third header</h1>
